components/body.js

```diff
@@ -0,0 +1,47 @@
+import Work from "./work.js";
+import Volunteer from "./volunteer.js";
+import Education from "./education.js";
+import Projects from "./projects.js";
+import Awards from "./awards.js";
+import Certificates from "./certificates.js";
+import Publications from "./publications.js";
+import Skills from "./skills.js";
+import Languages from "./languages.js";
+import Interests from "./interests.js";
+import References from "./references.js";
+import html from "../utils/html.js";
+
+export default function Body(resume) {
+    let resumeOrder = resume.meta.order
+    if (resumeOrder === undefined) {
+        resumeOrder = [
+            'work',
+            'volunteer',
+            'education',
+            'projects',
+            'awards',
+            'certificates',
+            'publications',
+            'skills',
+            'languages',
+            'interests',
+            'references',
+        ]
+    }
+
+    const orderComponentMap = {
+        work: Work(resume.work),
+        volunteer: Volunteer(resume.volunteer),
+        education: Education(resume.education),
+        projects: Projects(resume.projects),
+        awards: Awards(resume.awards),
+        certificates: Certificates(resume.certificates),
+        publications: Publications(resume.publications),
+        skills: Skills(resume.skills),
+        languages: Languages(resume.languages),
+        interests: Interests(resume.interests),
+        references: References(resume.references),
+    }
+
+    return html`${resumeOrder.map(section => html`${orderComponentMap[section]}`)}`;
+}
```

Comment:
We can simplify things a bit here by using the component map order as the default section order, while also deferring section rendering until it's needed (plus some renaming/null-checks):

```suggestion
const sectionComponentMap = {
  work: (resume) => Work(resume.work),
  volunteer: (resume) => Volunteer(resume.volunteer),
  education: (resume) => Education(resume.education),
  projects: (resume) => Projects(resume.projects),
  awards: (resume) => Awards(resume.awards),
  certificates: (resume) => Certificates(resume.certificates),
  publications: (resume) => Publications(resume.publications),
  skills: (resume) => Skills(resume.skills),
  languages: (resume) => Languages(resume.languages),
  interests: (resume) => Interests(resume.interests),
  references: (resume) => References(resume.references),
}

export default function Sections(resume) {
  const sections = resume.meta?.sections || Object.keys(sectionComponentMap)
  return html`${sections.map(section => sectionComponentMap[section]?.(resume))}`
}

```